extensions/ql-vscode/package.json

```
@@ -179,8 +179,8 @@
         },
         "codeQL.queryHistory.format": {
           "type": "string",
-          "default": "[%t] %q on %d - %s",
-          "description": "Default string for how to label query history items. %t is the time of the query, %q is the query name, %d is the database name, and %s is a status string."
+          "default": "%q on %d - %s, %r results [%t]",
```

Avoid pluralization issues:

```suggestion
          "default": "%q on %d - %s, %r result count [%t]",
```